setup.py

```diff
@@ -39,7 +39,7 @@
     },
     test_suite='tests',
     install_requires=[
-        'lxml<4.7.1',
+        'lxml<=4.9.1',
```

Comment:
Ideally, `python-saml` would not be pinned quite this tight. To address the CVE, maybe switch back to: 

```suggestion
        'lxml>=4.9.1',
```